.github/workflows/ci-cd.yml

```diff
@@ -59,41 +69,40 @@ jobs:
         id: build_projects
         shell: pwsh
         run: |
-          $projectsArray = @(
-            '.\src\Microsoft.OpenApi\Microsoft.OpenApi.csproj',
-            '.\src\Microsoft.OpenApi.Readers\Microsoft.OpenApi.Readers.csproj',
-            '.\src\Microsoft.OpenApi.Tool\Microsoft.OpenApi.Tool.csproj'
-          )
           $gitNewVersion = if ("${{ steps.tag_generator.outputs.new_version }}") {"${{ steps.tag_generator.outputs.new_version }}"} else {$null}
-          $projectCurrentVersion = ([xml](Get-Content .\src\Microsoft.OpenApi\Microsoft.OpenApi.csproj)).Project.PropertyGroup.Version
+          $projectCurrentVersion = ([xml](Get-Content -Path .\src\Microsoft.OpenApi\Microsoft.OpenApi.csproj)).Project.PropertyGroup.Version
           $projectNewVersion = $gitNewVersion ?? $projectCurrentVersion
 
-          $projectsArray | ForEach-Object {
-            dotnet build $PSItem `
-            -c Release # `
-            # -o $env:ARTIFACTS_FOLDER `
-            # /p:Version=$projectNewVersion 
+          Get-ChildItem -Path src/ -Filter *.csproj -Exclude *Workbench* -File -Recurse | ForEach-Object {
```

Comment:
Replace static array of project locations with a more elegant solution.

Reply:
out of curiosity, any reason why you don't simply target the sln?

Reply:
We just took the `build.cmd` file as an example and just never took a second thought about it, but now that you mention it unless there is a specific reason not to, we could change the build step to building only the solution instead 🙂

Reply:
We don't publish the Workbench tool, so I didn't bother building it for publishing nugets.  I don't have a problem building it as part of the pipeline.  However currently `dotnet build` can't build the Workbench project because of some error about the GenerateResource task.  Not sure what that is about.